script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt

```diff
@@ -141,57 +166,69 @@ The earth shakes. The atmosphere rumbles.
 [sha:06abc87603b6c0dd1d80b07954cd982ad17875e1]{
 -- Page 10, Offset 39375.
 -- 今まで観測された事のない魔力濃度。
-A concentration of magical energy like never seen before.
+The concentration of magical energy is utterly unprecedented.
 }
 [sha:359eed2acf97e335c8b61028d849cd2faf4bb60b]{
 -- Page 10, Offset 39376.
 -- 取りこんだ事象の熱量が極点にまで凝縮していく。
-The heat of the newfound phenomenon condenses to its peak.
+// 極点 lore term
+The heat of this novel phenomenon is condensing down to a single focality.
 }
 [sha:b6510739f09f863c347ef12be9fb99340681305d]{
 -- Page 11, Offset 39377.
 -- それは、言ってみれば原理の種だった。
-In a matter of speaking, it is a type of Principle.
+In a sense, it is the seed of a Principle.
 }
 [sha:37f4616ea0c8cbf5176b0dae501a688936215e64]{
 -- Page 11, Offset 39378.
 -- 10のマイナス35乗メートルの極点に圧縮された、この領域の全情報。
-All the information in this area is compressed into a single point of 10⁻³⁵ meters in diameter.
+// You know shit is getting serious when you drop some EXPONENTS
+// Credit though for getting the Planck length correct though, unless that's just coincidence
+// Game doesn't support any kind of normal exponent rendering (naturally) so added a custom glyph for ^-35
+// 極点 -> focality
+All of the information in this area has been compressed down to a single focality measuring just 10%{e_35} meters in diameter.
 }
 [sha:ad11066ff572807f9b39832d3884b8bd696f13bd]{
 -- Page 11, Offset 39379.
 -- 星の内海、概念次元にあるかぎりは何の異常も呼び起こさない天体の卵。
-The Inner-sea of the Planet; a Celestial Egg that raises no abnormalities for as long at it's within a conceptual dimension.
+// E G G
+The Inner Sea of the Planet;
+ a Celestial Egg that induces no abnormalities so long at it remains within the conceptual dimension.
 }
 [sha:1ffaec3647d06e88585b30fdba23eb06d82a1302]{
 -- Page 11, Offset 39380.
 -- だが、それが物質として出現してしまった時、膨大なエネルギーは極点では抑えきれなくなり、急速に膨張する。
-But once it manifests as matter, the vast amount of energy cannot be contained at its peak and rapidly expands.
+But as soon as it manifests in the physical world,
+ its boundless energy becomes unstable,
+ and rapidly expands from that focality.
 }
 [sha:2f71e4556b19b6f14e734c1f9b39d263342548a7]{
 -- Page 12, Offset 39381.
 -- 天文学において<膨張現象|インフレーション>と呼ばれる、
-In astronomy terms, it is something known as Inflation.
+// Tweaking this to be 'big bang' for the same reasons I did in 1G
+In astronomy, such an event is referred to as a Big Bang.
```

Comment:
I still prefer big bang, because it is _immediately_ clear what that means, and it is the common term for this phenomenon, but I feel like there may have been a later file that forced me to go back and grudgingly revert them all to inflation.